packages/tools/pie-css/test/scss/unit/settings/mq.spec.ts

```diff
@@ -0,0 +1,219 @@
+
+          .foo {
+            @include media('>=sm') {
+                text-align: center;
+            }
+          }
+        `;
+
+        const expectedCss = stripCSSWhitespace(`
+          @media (min-width: 600px) {
+            .foo {
+              text-align: center;
+            }
+          }
+        `);
+
+        // Act
+        const css = stripCSSWhitespace(compileCss(scssToTest));
+
+        // Assert
+        expect(css).toBe(expectedCss);
+    });
+    it('should render the md media query breakpoint', () => {
+        // Arrange
+        const scssToTest = `
+          @use 'settings' as *;
+
+          .foo {
+            @include media('>=md') {
+                text-decoration: underline;
+            }
+          }
+        `;
+
+        const expectedCss = stripCSSWhitespace(`
+          @media (min-width: 768px) {
+            .foo {
+              text-decoration: underline;
+            }
+          }
+        `);
+
+        // Act
+        const css = stripCSSWhitespace(compileCss(scssToTest));
+
+        // Assert
+        expect(css).toBe(expectedCss);
+    });
+    it('should render the lg media query breakpoint', () => {
+        // Arrange
+        const scssToTest = `
+          @use 'settings' as *;
+
+          .foo {
+            @include media('>=lg') {
+                text-transform: uppercase;
+            }
+          }
+        `;
+
+        const expectedCss = stripCSSWhitespace(`
+          @media (min-width: 1024px) {
+            .foo {
+              text-transform: uppercase;
+            }
+          }
+        `);
+
+        // Act
+        const css = stripCSSWhitespace(compileCss(scssToTest));
+
+        // Assert
+        expect(css).toBe(expectedCss);
+    });
+    it('should render the xl media query breakpoint', () => {
+        // Arrange
+        const scssToTest = `
+          @use 'settings' as *;
+
+          .foo {
+            @include media('>=xl') {
+                font-weight: bold;
+            }
+          }
+        `;
+
+        const expectedCss = stripCSSWhitespace(`
+          @media (min-width: 1280px) {
+            .foo {
+              font-weight: bold;
+            }
+          }
+        `);
+
+        // Act
+        const css = stripCSSWhitespace(compileCss(scssToTest));
+
+        // Assert
+        expect(css).toBe(expectedCss);
+    });
+    it('should render the xxl media query breakpoint', () => {
+        // Arrange
+        const scssToTest = `
+          @use 'settings' as *;
+
+          .foo {
+            @include media('>=xxl') {
+                font-style: italic;
+            }
+          }
+        `;
+
+        const expectedCss = stripCSSWhitespace(`
+          @media (min-width: 1440px) {
+            .foo {
+                font-style: italic;
+            }
+          }
+        `);
+
+        // Act
+        const css = stripCSSWhitespace(compileCss(scssToTest));
+
+        // Assert
+        expect(css).toBe(expectedCss);
+    });
```

Comment:
There is a lot of similar code in this file. Could we do some kind of `it.each` syntax to reduce code?

Reply:
Done!